src/Peering/Peering/help/New-AzPeeringDirectConnectionObject.md

```diff
@@ -129,6 +170,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MicrosoftProvidedIPAddress
+Who will provide the session address(es)?
```

Comment:
@rileymckenna this description should probably be changed to let the user know that by setting this switch parameter, Microsoft is providing the IP address

Reply:
yes i made the change, i guess it didnt update the documentation. its fixed.